### PR TITLE
allow overriding the popup background color

### PR DESCRIPTION
**Pull request by lduboeuf**
This allow to override default background color for popups

### CODE REVIEW

File: src/imports/Components/Popups/1.3/PopupBase.qml [hunk 1]
```diff
@@ -142,6 +142,14 @@ OrientationHelper {
       */
     property alias __dimBackground: background.dim
 
+    /*!
+      \internal
+      Property driving the dim color popup's background. The default is the same as
+      defined in the style
+      */
+    property alias __dimBackgroundColor: background.color
```
Comment (mardy) [inline src/imports/Components/Popups/1.3/PopupBase.qml:150]: Why not just `__backgroundColor`?

Comment (lduboeuf) [reply src/imports/Components/Popups/1.3/PopupBase.qml:150]: humm maybe to see difference between the dim background and the content bg ?.

Comment (mardy) [reply src/imports/Components/Popups/1.3/PopupBase.qml:150]: Ah, OK, I didn't know we were having two different backgrounds.

**Comment by dobey** [comment]
OK, this keeps failing in CI and I don't know why. Can you `git remote add upstream https://github.com/ubports/ubuntu-ui-toolkit` and then `git pull --rebase upstream xenial` and finally `git push -f` to update your branch? Thanks.

**Review by dobey — APPROVED** [review]
Looks OK. Thanks!

**Pull request merged**